14_final_schema.sql

```diff
@@ -73,6 +75,15 @@ USING(subject_id, hadm_id, stay_id, chart_hour)
 FULL JOIN
 `elevated-pod-307118.physionet.iv_antibiotics`
 USING(subject_id, hadm_id, stay_id, chart_hour)
+-- iv_abx_pharmacy is rolled up to the stay x chart_hour x medication grain 
+-- we need to roll it up further to the stay x chart_hour grain in order to join with the rest of the tables
+FULL JOIN
+(
+    select subject_id, hadm_id, stay_id, chart_hour, count(1) as abx_orders
+    FROM `elevated-pod-307118.physionet.iv_abx_pharmacy`
```

Comment:
Nit: At some point (not necessarily now), put the tablespace/schema (`elevated-pod-307118`) into a constant somewhere for later reference. That way it's only modified once.

Reply:
I agree! I've been getting by with DECLARE / SET / EXECUTE statements since I actually have to use a different schema from Rafiya. However, I _think_ that won't be necessary since after this previous work is done, we're most likely going to operating strictly from LifeBell's AWS Postgres DB as opposed to individual GCP schemas

Reply:
Sounds good!